content/blog/2020-announce-ticket-sales.md

```diff
@@ -0,0 +1,47 @@
+Title: PyCon India 2020 Tickets are going live soon! Grab yours before the special offer runs out!
+Date: 2020-08-16 17:00
+Author: Vipul Gupta (@vipulgupta2048)
+Category: 2020
+Slug: 2020-announce-ticket-sales
+Summary: Tickets for the conference is going live soon! Grab yours before we run all out of the early bird offers. 
+
+PyCon India is a grass-roots, volunteer-driven conference. What that means is every year for the past 12 years Python communities across the nation have come together to help plan, organize, manage, and run the conference. It's the country's largest premier conference for the Python programming language unparalleled in the opportunities, knowledge, and networking potential it fosters.
+
+With over 250+ proposals coming in globally, this year PyCon India 2020 is set to be the biggest one yet. The conference will be having a packed schedule of workshops, keynotes, talks, and dev-sprints strung together through 3 parallel tracks. Any person who has a conference ticket is eligible to attend the Devsprints. Although the Developer Sprints are free, we would be having registrations beforehand to manage the logistics.
+
+We will be starting with the ticket sales of our mega event soon and the details for the same will be available on [PyCon India official website](https://in.pycon.org). To make the event inclusive and accessible to all, we have kept the ticket prices to a bare minimum, namely.
+
+> **Rush to grab your early bird tickets now from [Townscript]()!**
```

Comment:
Here is the transcript page for PyCon India 2020 - https://www.townscript.com/e/pycon-india-2020

Reply:
Thanks for that. I was thinking I was missing something.